test/snapshot/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
```

Comment:
not sure we need checks DEPS for test code, I don't see chrome uses it

Reply:
run npm gn_check fails without this file